web/README.md

```
@@ -14,12 +14,53 @@ A **performance-first PWA** for solo location discovery, built with React Server
+cp .env.local.example .env.local
+# Add your OAuth credentials (see Authentication Setup below)
+
 pnpm dev                        # Start with Turbopack 🚀
 
 # App ready at http://localhost:3000
 # Lightning-fast HMR with Turbopack
 ```
 
+## 🔐 Authentication Status
+
+**✅ FULLY IMPLEMENTED**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)